docs/about/get-involved.md

```diff
@@ -5,7 +5,7 @@ We'd love for you to be part of our ever-growing community. There are a variety
 * **📖  Docs:** this is the place to learn how to build amazing sites with Frontity - oh look, you're already here! **😄**
 * **👨‍👩‍👧‍👦  Community forum:** join our forum and ask questions, share your knowledge, give us feedback and tell us how we're doing, and meet other cool Frontity people. We'd also love to know about what you're building with Frontity, so please do swing by the [forum](https://community.frontity.org/) and tell us about your projects.
 * **🐞  Contribute:** we use [GitHub](https://github.com/frontity/frontity) for bugs and pull requests. See our [Contributing](../contributing/README.md) section to find out how you can help develop Frontity, or improve this documentation.
-* **🗣  Social media**: interact with other Frontity users. Reach out to us on [Twitter](https://twitter.com/frontity).
-* 💌  **Newsletter:** do you want to receive the latest framework updates and news? Subscribe [here](https://frontity.org/#newsletter).
+* **🗣  Social media**: interact with other Frontity users. Reach out to us on [Twitter](https://twitter.com/frontity). Tweet about Frontity and what you're building with it using the hashtag **#Frontity**.
```

Comment:
I think it's better to mention us directly using `@frontity`. Something like...
Mention us in your tweets about Frontity and what you're building by using **@frontity**.